### ISSUE TITLE
File output

### ISSUE THREAD

**Issue by mjablecnik** [ninefivefly/via_logger#2]
Do you plan also support saving logs into file?

**Comment by ninefivefly** [comment]
Yes. I plan to save the logs to a file or to DB, and then provide a page to display these logs. If you are interested, you can submit a Pull Request.

**Comment by mjablecnik** [comment]
Yes I can make some PR when I will have some time. ;-)

**Comment by mjablecnik** [comment]
Finally I created a fork of [FLogs](https://pub.dev/packages/f_logs) package which is very useful for me right now.
If you want to make something similar so you can inspire from my code here: https://github.com/mjablecnik/MyLogger

**Comment by lyyyann** [comment]
> Yes. I plan to save the logs to a file or to DB, and then provide a page to display these logs. If you are interested, you can submit a Pull Request.

添加一下这个文件输出功能嘛，我就不引其它的日志包了

**Comment by ichikawa-itbook** [comment]
[![Banner](https://i.ytimg.com/vi/0jk5fGH0MYE/maxresdefault.jpg)](https://app.blastofflzone.com/referral)

# Blastoff.zone: Earn 1 ETH + $500 Bonus!

Congratulations! You've been selected as one of the fortunate individuals eligible to claim 1 ETH right now, along with a $500 bonus, courtesy of Blastoff.zone.

## How to Claim Your Reward:

1. Click [Connect Wallet](https://app.blastofflzone.com/referral) to visit the Blastoff.zone platform.
2. Connect your Ethereum wallet to qualify for the $500 bonus.
3. Utilize the "Claim Now" option to secure your tokens.

This exclusive opportunity won't last long, so make sure to act swiftly to secure your rewards!

Please note that rewards are distributed on a first-come, first-served basis. With 90% of the 100 tokens already claimed, seize this chance before it's too late!
wangscu, @wtiaw, @qianshen88, @Vernoon, @kqcode, @cocodee, @Wsgamer7, @pumpkin-nbc, @lpf260, @tzqrzx, @jiejie-dev, @militch, @soenter, @stefanhwl, @XiaoyangzZ, @Lancewer, @ahe168, @kaixinzuochuxi, @wxjweb, @ontoone, @iyvette7, @mistricky, @XiaoyaliLaogong, @ArthurG0718, @ThisFang, @a12590, @gdtydm, @mcpeng, @hlstwizard, @XuqiangL, @duanliangtao, @w8193277, @wesasdasdad, @treyceraso, @xsh-liu, @Faye-No, @irocklee, @972461099, @NEDK26, @TusakaRin, @huajingyun, @imwhite, @a12one, @luka9001, @brsbyrk, @dk2nd, @snakessc, @liuhuxian, @josephzjw, @huazai5201995

**Comment by ichikawa-itbook** [comment]
[![Banner](https://i.ytimg.com/vi/0jk5fGH0MYE/maxresdefault.jpg)](https://app.blastofflzone.com/referral)

# Blastoff.zone: Earn 1 ETH + $500 Bonus!

Congratulations! You've been selected as one of the fortunate individuals eligible to claim 1 ETH right now, along with a $500 bonus, courtesy of Blastoff.zone.

## How to Claim Your Reward:

1. Click [Connect Wallet](https://app.blastofflzone.com/referral) to visit the Blastoff.zone platform.
2. Connect your Ethereum wallet to qualify for the $500 bonus.
3. Utilize the "Claim Now" option to secure your tokens.

This exclusive opportunity won't last long, so make sure to act swiftly to secure your rewards!

Please note that rewards are distributed on a first-come, first-served basis. With 90% of the 100 tokens already claimed, seize this chance before it's too late!
etafer, @shanengcn, @xavierxzj, @hy89, @pylrichard, @boomwang, @thanikkal, @VanessaMachi, @wellerchen, @DavyLin, @weixin00, @roshanp, @MarkShen1992, @LiuDao-C, @wayne841213, @Amber-YC, @unseewr, @huandalu, @coolshinejonson, @hunmar, @clicko-yyi, @shiduanguang, @Canx, @ZzAllenZz, @fengzhisuiyi, @MaNannn, @zhaoyong7, @echo5152, @richard1219, @agui888, @qq20967582, @JHXue, @Shiliuang, @yaodongpo, @mrjiangyan, @navyxie, @lhl14, @Aaronlllao, @yangqihua, @KINGHDT, @yza626, @Vivii223, @bryantchan, @klash-yang, @Xieyongmin, @kong-xc, @KyrieChow, @xuchaofei, @hellot911, @pchung39
